### DISCUSSION TITLE
[Bug]:

### DISCUSSION THREAD

**Discussion by aavash209** [danny-avila/LibreChat#794] [Troubleshooting]
### Contact Details

kafleavash902@gmail.com

### What happened?

Invalid API key. Please check your API key and try again. You can do this by clicking on the model logo in the left corner of the textbox and selecting "Set Token" for the current selected endpoint. Thank you for your understanding.

### Steps to Reproduce

<img width="586" alt="image" src="https://github.com/danny-avila/LibreChat/assets/64249794/6af81419-fb40-4f78-bfe9-d4513f1bfe41">

### What browsers are you seeing the problem on?

Chrome

### Screenshots

<img width="586" alt="image" src="https://github.com/danny-avila/LibreChat/assets/64249794/a1c072c3-d84e-4934-a2f4-5b422af9f162">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by berry-13** [comment]
It's an error with the API key, please check if it is correct. Are you using a reverse proxy?

**Reply by danny-avila** [reply]
As mentioned there is an issue with how you configured this. Do check `.env.example` in the root of the project for more guidelines on api credentials @aavash209
